scripts/generateCrowdinConfig.js

```diff
@@ -0,0 +1,27 @@
+/*eslint-disable  @typescript-eslint/no-var-requires*/
```

Comment:
This is a script used in package.json to generate the `crowdin.yml` file, inserting the `.env` variables for `api_token` and `project_id`. The `crowdin.yml` file is used to upload and download content from the site.